sentry_sdk/serializer.py

```diff
@@ -118,6 +120,10 @@ def serialize(event, **kwargs):
     path = []  # type: List[Segment]
     meta_stack = []  # type: List[Dict[str, Any]]
 
+    do_not_trim_request_bodies = (
```

Comment:
Nitpicking: positive vars are easier to comprehend when used in conditions. Something like `should_trim_request_bodies`. (nitpicking stuff you can always decide to not change, and just go with your solution if you like it better)

Reply:
Makes sense, will change. Didn't like the name in the first place.

Reply:
Changed to `keep_request_bodies`